.github/workflows/code-climate-coverage-aggregation.yml

```diff
@@ -136,6 +136,11 @@ jobs:
               fi
           done
 
+      # Formatting the SHARED COMPONENTS coverage report
+      - name: Format the SHARED COMPONENTS coverage report
+        run: |
```

Comment:
you only need the | if you're doing multi line bash commands, but potato potatoe.